### ISSUE TITLE
“Saw” radical is not displaying correctly

### ISSUE THREAD

**Issue by igor10k** [davidsansome/tsurukame#661]
![IMG_6634](https://github.com/davidsansome/tsurukame/assets/1244342/94709d1f-b5c8-48d2-a17e-052e41917dac)

**Comment by UInt2048** [comment]
Duplicate of #658 

It has been fixed, but we’re waiting on App Store approval

**Comment by igor10k** [comment]
Ah, sorry, I searched the issues but didn’t find one as I had no idea this crashes the app.

**Comment by UInt2048** [comment]
@igor10k By the way, if you don’t want to wait for the App Store release, you can get the fix on TestFlight (build 981 or higher)
